.github/workflows/deploy-prod.yaml

```diff
@@ -1,13 +1,11 @@
 name: Deploy Prod
 
 on:
-  push:
-    tags:
-      - "v[0-9].[0-9]+.[0-9]+"
+  workflow_dispatch:
```

Comment:
This shouldn't be changing, this makes it a manual workflow and not triggered on tags

Reply:
Correction done